kratos/CMakeLists.txt

```diff
@@ -191,7 +201,7 @@ target_compile_definitions(KratosVersion PRIVATE
 
 ## Define library KratosCore to be included in all of the others
 add_library(KratosCore SHARED ${KRATOS_CORE_SOURCES} ${KRATOS_CORE_TESTING_ENGINE_SOURCES} ${KRATOS_CORE_INPUT_OUTPUT_SOURCES} ${KRATOS_TEST_SOURCES} $<TARGET_OBJECTS:KratosVersion>)
-target_link_libraries(KratosCore PUBLIC gidpost)
+target_link_libraries(KratosCore PUBLIC gidpost triangle)
```

Comment:
this is not related/conflicting with #5538?

Reply:
Nope, the files from #5538 are not even imported, linked. Can be removed safely